drawBot/context/baseContext.py

```diff
@@ -1777,7 +1780,7 @@ def appendGlyph(self, *glyphNames):
         # disable calt features, as this seems to be on by default
         # for both the font stored in the nsGlyphInfo as in the replacement character
         fontAttributes = {}
-        fontAttributes[CoreText.NSFontFeatureSettingsAttribute] = [openType.featureMap["calt_off"]]
+        fontAttributes[CoreText.kCTFontFeatureSettingsAttribute] = [dict(CTFeatureOpenTypeTag="calt", CTFeatureOpenTypeValue=False)]
```

Comment:
I guess if this needs to work on 10.10 the olde style font attributes must be added too.

Reply:
Hmm, this oddly enough seems to work as expected on 10.10.